packages/@aws-cdk/aws-backup/lib/rule.ts

```diff
@@ -185,6 +216,20 @@ export class BackupPlanRule {
       throw new Error(`'deleteAfter' must be between 1 and 35 days if 'enableContinuousBackup' is enabled, but got ${props.deleteAfter.toHumanString()}`);
     }
 
+    if (Array.isArray(props.copyActions) && props.copyActions.length > 0) {
```

Comment:
```suggestion
    if (props.copyActions && props.copyActions.length > 0) {
```

nit: not that you're wrong, but this is _maybe_ simpler?

Reply:
Good call, I'm all for making this simpler.